ext/serializer.c

```diff
@@ -1756,6 +1788,26 @@ void ddtrace_serialize_span_to_array(ddtrace_span_data *span, zval *array) {
         }
     }
 
+    if (inferred_span) {
+        zval *serialized_inferred_span = ddtrace_serialize_span_to_array(inferred_span, array);
+        zend_array *serialized_inferred_span_meta = Z_ARR_P(zend_hash_str_find(Z_ARR_P(serialized_inferred_span), ZEND_STRL("meta")));
+        zend_array *serialized_inferred_span_metrics = Z_ARR_P(zend_hash_str_find(Z_ARR_P(serialized_inferred_span), ZEND_STRL("metrics")));
+
+        zend_array *serialized_meta = Z_ARR_P(zend_hash_str_find(Z_ARR_P(el), ZEND_STRL("meta")));
+        zend_array *serialized_metrics = Z_ARR(metrics_zv);
+
+        transfer_data(serialized_metrics, serialized_inferred_span_metrics, ZEND_STRL("_dd.agent_psr"), true);
+        transfer_data(serialized_metrics, serialized_inferred_span_metrics, ZEND_STRL("_dd.rule_psr"), true);
+        transfer_data(serialized_metrics, serialized_inferred_span_metrics, ZEND_STRL("_dd.limit_psr"), true);
+
+        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("error.message"), false);
+        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("error.type"), false);
+        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("error.stack"), false);
+        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("error.ignored"), false);
```

Comment:
```suggestion
        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("error"), false);
        transfer_data(serialized_meta, serialized_inferred_span_meta, ZEND_STRL("track_error"), false);
```
I think `error.ignored` is not a thing after serialization, but `error` is.